server/template-versions/apple/1/pass.json

```
@@ -10,28 +10,28 @@
   "formatVersion": 1,
   "storeCard": {
     "headerFields": [
+      {
+        "key": "issueDate",
+        "label": "Issue Date",
+        "value": "2022-05-30"
+      },
       {
         "key": "id",
         "value": "not yet :)",
-        "label": "Passport ID"
+        "label": "Number"
       }
     ],
     "primaryFields": [],
     "secondaryFields": [
       {
         "key": "holder",
-        "label": "Holder",
+        "label": "Citizen Name",
```

```suggestion
        "label": "Passport Holder",
```
This was my fault, the design on figma was outdated, on the passport token we are using this label.